packages/iobroker.vis-2/src-vis/src/Vis/Widgets/Basic/BasicViewInWidget8.tsx

```diff
@@ -53,7 +53,7 @@ class BasicViewInWidget8 extends VisRxWidget<RxData> {
         }
 
         const viewArr = [];
-        let i = 0;
+        let i = 1;
```

Comment:
at least during my tests first "index" was always 1 even though set to 0 in widget info